Content.Client/Content.Client.csproj

```diff
@@ -37,4 +37,12 @@
 
   <Import Project="..\RobustToolbox\MSBuild\Robust.Properties.targets" />
   <Import Project="..\RobustToolbox\MSBuild\XamlIL.targets" />
+  <ItemGroup>
+    <EmbeddedResource Remove="_nf\m_emp\M_EmpMenu.xaml" />
+  </ItemGroup>
+  <ItemGroup>
+    <AdditionalFiles Update="_NF\M_Emp\M_EmpMenu.xaml">
+      <Generator>MSBuild:Compile</Generator>
+    </AdditionalFiles>
+  </ItemGroup>
```

Comment:
the curse of auto generated xaml for new UIs.  This all can get dropped in the csproj file